application/frontend/controllers/SiteController.php

```diff
@@ -40,14 +40,22 @@ public function actionStatus()
             Yii::error('DB connection problem: ' . $e->getMessage());
             throw new Http500('DB connection problem.', $e->getCode());
         }
-        
+
+        // Checking the email service status causes too many error emails to be sent out to the dev team
+        // $this->checkEmailServiceStatus($webApp);
+
+        Yii::$app->response->statusCode = 204;
+    }
+
+    private function checkEmailServiceStatus($webApp)
+    {
         try {
             $emailer = $webApp->get('emailer');
         } catch (Exception $e) {
             Yii::error('Emailer config problem: ' . $e->getMessage());
             throw new Http500('Emailer config problem.');
```

Comment:
You may still want to check for Emailer config problems, just not the `$emailer->getSiteStatus()`.

Reply:
:+1: